sambacc/ctdb.py

```diff
@@ -338,9 +338,23 @@ def manage_nodes(
     pause_func: typing.Callable,
```

Comment:
`manage_nodes()` now exists for testing purpose only, right?

Reply:
Yeah, there are now a bunch of functions that are consumed only by the test suite. It's a TODO item to circle back and update the tests and eventually remove the functions.